### PR TITLE
add profile

### PR DESCRIPTION
**Pull request by PacoGD**

### CODE REVIEW

**Comment by EzequielBurgosL** [comment]
Well done! keep up the good job!

File: staff/PacoGD/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# [Github](https://github.com/PacoGD/)
```
Comment (manuelbarzi) [inline staff/PacoGD/README.md]: ADD # Shares

Comment (EzequielBurgosL) [inline staff/PacoGD/README.md]: ADD # shares

Comment (PacoGD) [reply staff/PacoGD/README.md]: Done

**Comment by manuelbarzi** [comment]
+1

**Pull request merged**